app/controllers/galleries_controller.rb

```diff
@@ -16,7 +17,6 @@ def index
     @hero_image = homepage_hero_image
 
     @body_cache_key = foyer_body_cache_key(topic: @selected_topic, per: @galleries.limit_value, page: @galleries.current_page)
-    @documents = search_api_for_image_metadata(images) unless body_cached?
```

Comment:
Removing this line and constructing the gallery tiles/images structure in the view via the new helper means we don't need to assign images on line :16 also that will remove the codeclimate issue on this PR.
plus we can remove the 'gallery_images_for_foyer' method from the controller.